### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by prrajput1199**
What type of PR is this?
/kind documentation

What this PR does / why we need it:
Fixes the documentation

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:
Minor changes in documentation for "addon"" to "add-on"

Does this PR introduce a user-facing change?

### CODE REVIEW

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Till now we have only accessed the nginx server from within the cluster. Before
 * An nginx server configured to use the certificates
 * A [secret](/docs/concepts/configuration/secret/) that makes the certificates accessible to pods
 
-You can acquire all these from the [nginx https example](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/). This requires having go and make tools installed. If you don't want to install those, then follow the manual steps later. In short:
+You can acquire all these from the [nginx https example](https://github.com/kubernetes/examples/tree/master/staging/https-nginx/). This requires having to go and make tools installed. If you don't want to install those, then follow the manual steps later. In short:
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/services/connect-applications-service.md:236]: @prrajput1199 this is not a valid change.

File: content/en/docs/tutorials/services/connect-applications-service.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ KUBERNETES_SERVICE_PORT_HTTPS=443
 
 ### DNS
 
-Kubernetes offers a DNS cluster addon Service that automatically assigns dns names to other Services. You can check if it's running on your cluster:
+Kubernetes offers a DNS cluster add-on Service that automatically assigns dns names to other Services. You can check if it's running on your cluster:
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/services/connect-applications-service.md:196]: Trivial edit.

**Review by divya-mohan0209 — COMMENTED** [review]
@prrajput1199 These are a bunch of trivial edits and changes that aren't valid. We understand that you're enthusiastic about contributing to Kubernetes and open source, however this is the fourth PR that I'm seeing with a similar pattern of edits. Please read the contributing guide and our trivial edits policy before submitting a PR.

**Comment by amolmote** [comment]
Hi @prrajput1199 , you can start from this issue #35600.

**Comment by reylejano** [comment]
Hi @prrajput1199 ,
I'm closing this PR because a reviewer disagrees with the changes proposed in this PR and I agree the changes proposed are not correct and this PR needs rebase -- otherwise there is a merge conflict.
If you feel otherwise feel free to reopen the PR, rebase, and we can continue the discussion
/close

**Pull request closed without merging**